src/darwin/Framework/CHIP/CHIPOperationalCredentialsDelegate.mm

```diff
@@ -272,6 +293,14 @@ static BOOL isRunningTests(void)
     chip::Crypto::P256PublicKey pubkey;
     ReturnErrorOnFailure(chip::Crypto::VerifyCertificateSigningRequest(csr.data(), csr.size(), pubkey));
 
+    uint16_t dacVid = VendorId::NotSpecified;
+    uint16_t dacPid = 0;
+    ReturnErrorOnFailure(chip::Crypto::ExtractDNAttributeFromX509Cert(MatterOid::kVendorId, DAC, dacVid));
+    ReturnErrorOnFailure(chip::Crypto::ExtractDNAttributeFromX509Cert(MatterOid::kProductId, DAC, dacPid));
+
+    chip::CATValues assignedCATs;
+    ReturnErrorOnFailure(GetDeviceCATs(static_cast<VendorId>(dacVid), dacPid, assignedCATs));
```

Comment:
Why are CATs a function of vendor id and product id, exactly?

Reply:
That was my way of thinking that CATs for devices should be a function of `VID` and `PID` values.

For example one of the potential CAT categories will be assigned to all devices that can function as `admins` in the ecosystem. These devices will be recognized by the commissioner using their `PID` values (possibly also `VID`).

Another CAT category can be assigned to all light-switches, which will be also recognized by the commissioner using device `PID`.

Reply:
Let's level set though.  It is literally impossible for sdk development to predict what logic a vendor might use to assign CATs.

We therefore need to make sure that any logic along those lines only exists in the concrete operational credential delegate examples.  Such logic cannot e.g. exist in the OperationalCredentialsDelegate interface.

It appears this PR is honoring that.  But I think it's important to highlight this.

Reply:
I'll create a separate ticket to follow up with proper implementation.